plugins/background_fetch/configs.cc

```diff
@@ -44,8 +44,13 @@ BgFetchConfig::readConfig(const char *config_file)
     snprintf(file_path, sizeof(file_path), "%s/%s", TSInstallDirGet(), config_file);
     file = TSfopen(file_path, "r");
     if (nullptr == file) {
-      TSError("[%s] invalid config file", PLUGIN_NAME);
-      return false;
+      TSDebug(PLUGIN_NAME, "Failed to open config file %s, trying config path", config_file);
+      snprintf(file_path, sizeof(file_path), "%s/%s", TSConfigDirGet(), config_file);
+      file = TSfopen(file_path, "r");
+      if (NULL == file) {
+        TSError("[%s] invalid config file", PLUGIN_NAME);
+        return false;
+      }
```

Comment:
This should be:
```C
if (*config_file == '/') {
   file = TSfopen(config_file, "r");
} else {
  file = path.join(TSConfigDirGet(), config_file);
}
```

By trying the given path without qualification you allow operators to accidentally depend on the current working directory. The pseudocode above is what all the other plugins do, and I think that it is worth being consistent with this.